src/librustc/ich/hcx.rs

```diff
@@ -281,7 +281,7 @@ impl<'a> ToStableHashKey<StableHashingContext<'a>> for ast::NodeId {
     }
 }
 
-impl<'a> HashStable<StableHashingContext<'a>> for Span {
```

Comment:
I'd like this impl to be moved to `syntax_pos`. Add a method on `syntax_pos::StableHashingContextLike` to get the source map to allow this.